tests/commands/test_commands.py

```diff
@@ -42,22 +40,21 @@ def test_main_entrypoint(mock_start):
     assert mock_start.called
 
 
-def test_bigchain_run_start(mock_run_configure,
-                            mock_processes_start,
-                            mock_db_init_with_existing_db,
-                            mocked_setup_logging):
-    from bigchaindb import config
+@patch('bigchaindb.log.setup_logging')
+@patch('bigchaindb.commands.bigchaindb._run_init')
+@patch('bigchaindb.config_utils.autoconfigure')
+def test_bigchain_run_start(mock_setup_logging, mock_run_init,
+                            mock_autoconfigure, mock_processes_start):
     from bigchaindb.commands.bigchaindb import run_start
     args = Namespace(config=None, yes=True,
                      skip_initialize_database=False)
     run_start(args)
-    mocked_setup_logging.assert_called_once_with(user_log_config=config['log'])
+    assert mock_setup_logging.called
```

Comment:
What's the motivation for the weaker assertion?

Reply:
The removed line does not work like this anymore. Before `setup_logging` had parameters you could provide but that's not the case anymore.